### PR TITLE
[LWIMIQ] The school year begins - let's improve our GPA

### PR DESCRIPTION
**Pull request by skateman**

### CODE REVIEW

**Comment by JPrause** [comment]
@imtayadeway @jntullo @syncrou can you review.

File: site/_posts/2017-09-04-the-school-year-begins.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+published: true
+tags: LWIMIQ
+---
+
+Hello, [Dávid](https://github.com/skateman) here from the UI team. Let's have a quick recap what happened in the last week on the magical land of ManageIQ.
+
+Honestly, I gave up on counting how many repositories we are currently maintaining. The only number I come up with is ~220 and this is the number of PRs we merged last week. Around 50% of these PRs got merged into [core][manageiq PRs merged] and into the [two][manageiq-ui-classic PRs merged] [UI][[manageiq-ui-service PRs merged]] repos, feel free to check them out.
+
+## Featured
+[Martin](https://github.com/martinpovolny) (but not [Martin](https://github.com/himdel) and [Martin](https://github.com/hyperkid123)) decided to increase the [Code Climate GPA](https://codeclimate.com/github/ManageIQ/manageiq-ui-classic) of the UI Classic repo. I lost counting of the PRs, but the outcome is [visible](https://codeclimate.com/github/ManageIQ/manageiq-ui-classic/trends/gpa).
```
Comment (jntullo) [inline site/_posts/2017-09-04-the-school-year-begins.md]: perhaps "I lost count of the PRs"?

File: site/_posts/2017-09-04-the-school-year-begins.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+---
+title: "Last Week in ManageIQ: The school year begins - let's improve our GPA"
+author: skateman
+date: 2017-09-05
+comments: true
+published: true
+tags: LWIMIQ
+---
+
+Hello, [Dávid](https://github.com/skateman) here from the UI team. Let's have a quick recap what happened in the last week on the magical land of ManageIQ.
```
Comment (jntullo) [inline site/_posts/2017-09-04-the-school-year-begins.md]: maybe a "quick recap about what happened"

**Comment by skateman** [comment]
@jntullo both fixed.

File: site/_posts/2017-09-04-the-school-year-begins.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+author: skateman
+date: 2017-09-05
+comments: true
+published: true
+tags: LWIMIQ
+---
+
+Hello, [Dávid](https://github.com/skateman) here from the UI team. Let's have a quick recap about what happened in the last week on the magical land of ManageIQ.
+
+Honestly, I gave up on counting how many repositories we are currently maintaining. The only number I come up with is ~220 and this is the number of PRs we merged last week. Around 50% of these PRs got merged into [core][manageiq PRs merged] and into the [two][manageiq-ui-classic PRs merged] [UI][[manageiq-ui-service PRs merged]] repos, feel free to check them out.
```
Comment (imtayadeway) [inline site/_posts/2017-09-04-the-school-year-begins.md]: Think there are too many `[` and `]` in that last link which is causing it to fail to render a link!

File: site/_posts/2017-09-04-the-school-year-begins.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+## Featured
+[Martin](https://github.com/martinpovolny) (but not [Martin](https://github.com/himdel) and [Martin](https://github.com/hyperkid123)) decided to increase the [Code Climate GPA](https://codeclimate.com/github/ManageIQ/manageiq-ui-classic) of the UI Classic repo. I lost count of the PRs, but the outcome is [visible](https://codeclimate.com/github/ManageIQ/manageiq-ui-classic/trends/gpa).
+
+## Improved
+The Service UI guys hired a [new guy](https://github.com/apps/greenkeeper) for keeping their dependencies up to date and he is doing a [suspiciously good job](https://github.com/ManageIQ/manageiq-ui-service/pulls?q=is%3Apr+is%3Amerged+base%3Amaster+merged%3A%222017-08-28+..+2017-09-03%22+sort%3Acreated-desc+author%3Aapp%2Fgreenkeeper). I am pretty sure that he will never lose his job because of the upcoming automatization and robots everywhere.
```
Comment (imtayadeway) [inline site/_posts/2017-09-04-the-school-year-begins.md]: Perhaps a gender-neutral "folks" or "people"/"person" would be better here?

File: site/_posts/2017-09-04-the-school-year-begins.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+## Fixed
+Brian is our maintainer of remote consoles. His task this week was to [fix](https://github.com/ManageIQ/manageiq/pull/15901) the web-based Cockpit console for multiple providers. Meanwhile [Joe](https://github.com/jvlcek) fixed a possible user duplication issue in the external authentication by [assigning UUIDs](https://github.com/ManageIQ/manageiq/pull/15535).
+
+## New
+What if I told you that [we're using Angular](https://github.com/ManageIQ/manageiq-ui-classic/pull/1989) in all tree views on the left side of the screen? Actually it's not 100% accurate, but let's say that there is just a single ~~Gallic tribe~~ screen that ~~the Romans were~~ I was not able to ~~conquer~~ convert.
+Also one of the container guys, [Nimrod](https://github.com/nimrodshn) added some [new features](https://github.com/ManageIQ/manageiq-ui-classic/pull/1840) into the container image list view.
```
Comment (imtayadeway) [inline site/_posts/2017-09-04-the-school-year-begins.md]: Similarly here - prefer "folks"/"people" to "guys"

**Comment by imtayadeway** [comment]
@skateman love this issue! LGTM with a couple of suggestions above

**Comment by JPrause** [comment]
@skateman let me and @cybette know when all edits are complete. Then we'll see about merging this puppy. 🐩

**Comment by skateman** [comment]
@imtayadeway fixed, @JPrause @cybette ready to merge!

File: site/_posts/2017-09-04-the-school-year-begins.md (hunk 4 above)
Comment (skateman) [reply site/_posts/2017-09-04-the-school-year-begins.md]: @imtayadeway thanks, there's no such thing as gender in my language so for me everything sounds like gender-neutral :wink:

File: site/_posts/2017-09-04-the-school-year-begins.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+## Featured
+[Martin](https://github.com/martinpovolny) (but not [Martin](https://github.com/himdel) and [Martin](https://github.com/hyperkid123)) decided to increase the [Code Climate GPA](https://codeclimate.com/github/ManageIQ/manageiq-ui-classic) of the UI Classic repo. I lost count of the PRs, but the outcome is [visible](https://codeclimate.com/github/ManageIQ/manageiq-ui-classic/trends/gpa).
+
+## Improved
+The Service UI folks hired a [new guy](https://github.com/apps/greenkeeper) for keeping their dependencies up to date and he is doing a [suspiciously good job](https://github.com/ManageIQ/manageiq-ui-service/pulls?q=is%3Apr+is%3Amerged+base%3Amaster+merged%3A%222017-08-28+..+2017-09-03%22+sort%3Acreated-desc+author%3Aapp%2Fgreenkeeper). I am pretty sure that he will never lose his job because of the upcoming automatization and robots everywhere.
```
Comment (imtayadeway) [inline site/_posts/2017-09-04-the-school-year-begins.md]: One last nit (sorry) - greenkeeper still has a gender. Could we change that "guy" to one of "person"/"developer"/"engineer"/"programmer"/etc and s/he/they/ ? I really ❤️ this section though - it's so charming!

Comment (JPrause) [reply site/_posts/2017-09-04-the-school-year-begins.md]: ? I think it's okay to say he/she him/her,....
However, I agree,.. guy/gal is too friendly.
But, I'll leave it to David to decide.

Comment (cybette) [reply site/_posts/2017-09-04-the-school-year-begins.md]: or maybe just "hired someone new" and use "it". 

(side note, there's no gender in Finnish either which makes things like this a lot easier. but everything else about Finnish is hard :stuck_out_tongue: )

File: site/_posts/2017-09-04-the-school-year-begins.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+
+## Fixed
+Brian is our maintainer of remote consoles. His task this week was to [fix](https://github.com/ManageIQ/manageiq/pull/15901) the web-based Cockpit console for multiple providers. Meanwhile [Joe](https://github.com/jvlcek) fixed a possible user duplication issue in the external authentication by [assigning UUIDs](https://github.com/ManageIQ/manageiq/pull/15535).
```
Comment (cybette) [inline site/_posts/2017-09-04-the-school-year-begins.md]: Link to Brian's github page for consistency?

**Comment by skateman** [comment]
Fixed again

**Pull request merged**